<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathPlanning</class>
 <widget class="QMainWindow" name="PathPlanning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PathPlanning</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frameViewer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButtonReset">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>480</y>
         <width>201</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>Display options</string>
       </property>
       <widget class="QCheckBox" name="checkBoxColModel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Collision model</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxRobot">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>robot</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxObstacles">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>obstacles</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxFloor">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>floor</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxOption1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>option 1</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>170</y>
         <width>201</width>
         <height>301</height>
        </rect>
       </property>
       <property name="title">
        <string>Control</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Trajectory</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumberTrajectory">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>30</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderTrajectory">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="labelMinPos">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelMaxPos">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>90</y>
          <width>41</width>
          <height>20</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>100</string>
        </property>
       </widget>
       <widget class="QPushButton" name="visualizeGridButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>171</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Visualize Grid (PF only)</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonStep2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>171</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Step 2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonStep3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>171</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Step 3</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonRunPlanner">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>171</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Run Planner</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>680</y>
         <width>201</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Info</string>
       </property>
       <widget class="QLabel" name="labelInfo1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>181</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelInfo2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>191</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelInfo3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>191</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Select Scene</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxScene">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Select Algorithm</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxAlgo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1103</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
